src/Orleans.Runtime/Hosting/DefaultSiloServices.cs

```diff
@@ -76,32 +74,12 @@ internal static void AddDefaultServices(HostBuilderContext context, IServiceColl
             services.TryAddTransient<ConsistentRingQueueBalancer>();
             services.TryAddTransient(typeof(IStreamSubscriptionObserver<>), typeof(StreamSubscriptionObserverProxy<>));
 
-            services.TryAddSingleton<ProviderManagerSystemTarget>();
-            services.TryAddSingleton<StatisticsProviderManager>();
-            services.AddFromExisting<IProviderManager, StatisticsProviderManager>();
-
-            // storage providers
-            services.TryAddSingleton<StorageProviderManager>();
-            services.AddFromExisting<IProviderManager, StorageProviderManager>();
-            services.TryAddFromExisting<IKeyedServiceCollection<string, IStorageProvider>, StorageProviderManager>(); // as named services
-            services.TryAddSingleton<IStorageProvider>(sp => sp.GetRequiredService<StorageProviderManager>().GetDefaultProvider()); // default
-
-            // log concistency providers
-            services.TryAddSingleton<LogConsistencyProviderManager>();
-            services.AddFromExisting<IProviderManager, LogConsistencyProviderManager>();
-            services.TryAddFromExisting<IKeyedServiceCollection<string, ILogConsistencyProvider>, LogConsistencyProviderManager>(); // as named services
-            services.TryAddSingleton<ILogConsistencyProvider>(sp => sp.GetRequiredService<LogConsistencyProviderManager>().GetDefaultProvider()); // default
-
-            services.TryAddSingleton<BootstrapProviderManager>();
-            services.AddFromExisting<IProviderManager, BootstrapProviderManager>();
-            services.TryAddSingleton<LoadedProviderTypeLoaders>();
+            services.TryAddSingleton<FallbackSystemTarget>();
+            services.TryAddSingleton<SiloUnobservedExceptionsHandler>();
```

Comment:
why are we adding `SiloUnobservedExceptionsHandler` back? it was made optional , turned off by default in #3829

Reply:
Likely I missed this in rebase/merge.  Will remove.  Thanks, good catch.